composer.json

```diff
@@ -74,4 +77,4 @@
             "Sonata\\BlockBundle\\Tests\\": "tests/"
         }
     }
-}
+},
```

Comment:
Remove comma

Reply:
I saw it after request for review. Fixed.